env.d/development/common

```diff
@@ -42,3 +42,11 @@ DJANGO_EMAIL_PORT=1025
 
 # Backoffice
 JOANIE_BACKOFFICE_BASE_URL="http://localhost:8072"
+
+# Signature backend
+JOANIE_SIGNATURE_BACKEND = "joanie.signature.backends.lex_persona.LexPersonaClient"
+JOANIE_SIGNATURE_LEXPERSONA_BASE_URL = "https://fake-base-url.test"
+JOANIE_SIGNATURE_LEXPERSONA_CONSENT_PAGE_ID = "fake_cop_id"
+JOANIE_SIGNATURE_LEXPERSONA_SESSION_USER_ID = "fake_user_id"
+JOANIE_SIGNATURE_LEXPERSONA_PROFILE_ID = "fake_profile_id"
+JOANIE_SIGNATURE_LEXPERSONA_TOKEN = "fake_token_id"
```

Comment:
I'll comment those variables and leave values empty like Badge providers

Reply:
Since we have a method in the signature app ( `joanie.core.signature.apps.py` ) that verifies if there is a signature provider declared, I have to comment the whole block in `common` file, such as : 
```
# JOANIE_SIGNATURE_BACKEND = "joanie.signature.backends.lex_persona.LexPersonaClient"
# JOANIE_SIGNATURE_LEXPERSONA_BASE_URL = "https://fake-base-url.test"
# JOANIE_SIGNATURE_LEXPERSONA_CONSENT_PAGE_ID =
# JOANIE_SIGNATURE_LEXPERSONA_SESSION_USER_ID =
# JOANIE_SIGNATURE_LEXPERSONA_PROFILE_ID =
# JOANIE_SIGNATURE_LEXPERSONA_TOKEN =
```
If I let `JOANIE_SIGNATURE_BACKEND` uncommented, it will raise an error when we build the application : 
```
django.core.exceptions.ImproperlyConfigured: Required setting JOANIE_SIGNATURE_LEXPERSONA_BASE_URL is not defined.
```